### PR TITLE
Update manual testing guide for addtag command

### PR DESCRIPTION
**Pull request by blackpanther9229**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -559,6 +559,22 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Adding a tag
+1. Deleting a tag while all tags are being shown
```
Comment (CYX22222003) [inline docs/DeveloperGuide.md:563]: should it be adding

**Review by CYX22222003 — DISMISSED** [review]

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -588,12 +588,29 @@ testers are expected to do more *exploratory* testing.
...
+   1. Test case: `addtag 2 t/homie t/homie`
+      Expected: The second person now has 3 tags `floortball`, `friends` and `homie`. 
+      
+   1. With the following test case:
+      1. `addtag 1 t/CS2040s`
+      1. Test case: `addtag 1 t/CS2030s t/CS2040S`
+      1. Test case: `addtag 0 t/volleyball` <br>
+      1. Test case: `addtag 3 t/homie` <br>
+      1. Test case: `addtag 2` <br>
+         Expected: No new tag added. Error details shown in the status message. All status bar remain the same.
```
Comment (KrashKart) [inline docs/DeveloperGuide.md:608]: There is no status bar.

**Review by KrashKart — CHANGES_REQUESTED** [review]
LGTM! Just some minor changes

**Review by KrashKart — APPROVED** [review]
LGTM!

**Pull request merged**